package.json

```diff
@@ -80,18 +80,6 @@
     "url": "https://github.com/mojaloop/thirdparty-sdk/issues"
   },
   "homepage": "https://github.com/mojaloop/thirdparty-sdk#readme",
-  "@comment dependencies": [
-    "koa2-oauth-server is an old unmaintained repo. It uses the now unsupported git protocol.",
```

Comment:
Is this no longer applicable?

Reply:
Yes, @mdebarros. `koa2-oauth-server` has removed from sdk-scheme-adapter.